src/content/whats-new/2022/11/whats-new-11-18-extended-retention-data-plus-upgrade.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Extend data retention in-product for compliance and longer-term analysis'
+summary: 'Our new in-product capability makes it easy for you to adjust your retention or access Data Plus without needing to contact sales'
+releaseDate: '2022-11-22'
+learnMoreLink: 'https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-retention/#adjust-retention'
+getStartedLink: 'https://one.newrelic.com/data-management-hub'
+---
+
+## Extending data retention and upgrading to Data Plus has become simpler
+
+We've made it easy to extend your data retention directly in-product, giving you the controls to decide how long you need your data for compliance or performing long-term analysis. Extended retention only costs $0.05/GB per month, giving you the ability to conduct historical analysis to get deeper insights than you could with standard retention. [Learn more about data retention for specific data types.](https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-retention/#retention-periods)
+
+Also included in this release is the ability to upgrade to [Data Plus](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus) in-product, which includes 90 days of extended retention plus and so much more (see, that's why we call it "Plus"). Data Plus also gives you up to 3x the maximum limit on number of queries and 10x the maximum query duration, streaming and historical data export, logs obfuscation, and HIPAA and FEDRamp compliance, with more enterprise-grade capabilities coming soon.
+
+## How to get started
+
+1. Extend data retention or upgrade to Data Plus from the [data management hub](https://one.newrelic.com/data-management-hub).
+2. Learn more about [data retention](https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-retention/#retention-periods).
+3. If you have further questions, reach out to your account team or check out our [pricing page](https://newrelic.com/pricing).
+
```

Comment:
Currently EU data centers do not have the in-product extended retention and Data Plus upgrade.

Reply:
@bradleycamacho since Emory confirmed this is good to go, I merged, but maybe follow up with him on this and see if he wants to add it as a note somewhere.

Reply:
Add to line 17:

Note: We currently have not enabled this feature for EU based accounts yet, but it will be released soon after.

Reply:
I confirmed with @emoryzhao that we no longer need this EU reference since there's already this: 

(Note that [this feature isn't available to all organizations](https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-retention#requirements).)